lib/msal-browser/test/interaction_client/RedirectClient.spec.ts

```diff
@@ -3604,30 +3634,33 @@ describe("RedirectClient", () => {
                         correlationId: RANDOM_TEST_GUID,
                         postLogoutRedirectUri: TEST_URIS.TEST_REDIR_URI,
                     };
-                    expect(logoutUriSpy.calledWith(validatedLogoutRequest));
+                    expect(logoutUriSpy).toHaveBeenCalledWith(
+                        validatedLogoutRequest
+                    );
```

Comment:
This `expect` never resolves, and causes the test to fail.

Reply:
Same comment here r.e. something else is failing first.

Reply:
Actually on second glance, you have two `done` calls in this test. It's likely exiting on the other one before it reaches these.

Reply:
I think the test already had the two `done` calls before I touched it?

Reply:
Could be that the difference between sinon/jest mocks alters the timing and is bringing attention to a pre-existing problem. I'd start by picking one or the other to remove.